x-pack/plugins/security_solution/cypress/e2e/detection_rules/rules_selection.cy.ts

```diff
@@ -46,7 +46,9 @@ describe('Rules selection', () => {
 
     cy.get(SELECT_ALL_RULES_BTN).click();
 
-    cy.get(SELECTED_RULES_NUMBER_LABEL).should('contain.text', totalNumberOfPrebuiltRules);
+    getPrepackagedRulesCount().then((prepackagedRulesCount) => {
+      cy.get(SELECTED_RULES_NUMBER_LABEL).should('contain.text', prepackagedRulesCount);
+    });
```

Comment:
Yeah, I can see that `loadPrebuiltDetectionRules` is used here in the tests for rule selection in the table, and I don't see how that feature is related to prebuilt rules and why we should be loading all of them to test it.

Seems like we'll need to revisit e2e test coverage for the table after we refactor it.